src/test/java/software/amazon/smithy/lsp/ext/SmithyProjectTest.java

```diff
@@ -51,4 +51,25 @@ public void walkingImports() throws Exception {
         }
 
     }
+
+    @Test
+    public void respectingEmptyConfig() throws Exception {
```

Comment:
Just to test the behaviour we expect - empty config means everything is discovered